lib/improver/orographic_enhancement.py

```diff
@@ -509,44 +503,25 @@ def _create_output_cubes(self, orogenh_data, reference_cube):
             except KeyError:
                 continue
 
-        orogenh = iris.cube.Cube(
+        orog_cube = iris.cube.Cube(
```

Comment:
There's a reason this was called `orogenh` (orographic enhancement) rather than `orog` (orography height)...

Reply:
change to  `orog_enhance_cube`